DashAI/back/core/schema_fields/int_field.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional
+
+from pydantic import Field
+from typing_extensions import Annotated
+
+
+def int_field(
```

Comment:
Maybe here you could try beartype to make sure that the types that are being sent to this function are the ones that should be being sent (runtime type checking).

```suggestion
# from beartype import beartype
# ...
@beartype
def int_field(
```